app/viewmodels/models/RadioListViewModel.scala

```diff
@@ -64,4 +64,7 @@ object RadioListRowViewModel {
 
   def apply(content: Message, value: String, hint: Message): RadioListRowViewModel =
     RadioListRowViewModel(content, value, Some(hint))
+
+  def apply(content: Message): RadioListRowViewModel =
```

Comment:
You need a value for the radio list item, how will the selected answer ne matched to the corresponding ADT value?